src/Components/EnvironmentPicker/EnvironmentPickerManager.ts

```diff
@@ -141,6 +146,7 @@ export const getContainerNameFromUrl = (containerUrl: string) => {
 };
 
 export const getContainerName = (container: string | IAzureResource) => {
+    // container can either be the name string of the container directly or the azure resource object
     return typeof container === 'string' ? container : container?.name;
 };
```

Comment:
![image](https://user-images.githubusercontent.com/57726991/186237063-ab4f9a01-ef26-42ed-811e-4ae122bfc7e6.png)

Reply:
had commented and marked as "resolved" this on the original PR